opentelemetry-otlp/tests/grpc_build.rs

```diff
@@ -44,13 +44,17 @@ fn build_content_map() -> HashMap<PathBuf, String> {
     let dict =
         std::fs::read_dir("src/proto/grpcio").expect("cannot open dict of generated grpc files");
 
-    for entry in dict.into_iter().flatten() {
-        map.insert(
-            entry.path(),
-            std::fs::read_to_string(entry.path()).unwrap_or_else(|_| {
-                panic!("cannot read from file {}", entry.path().to_string_lossy())
-            }),
-        );
-    }
+    let _ = dict
+        .into_iter()
+        .flatten()
+        .map(|entry| {
```

Comment:
I meant you could collect into `map` directly by yielding `(entry.path(), std::fs::read_to_string())` items.